### DISCUSSION TITLE
Use specified audio bitrate only if source audio bitrate is higher

### DISCUSSION THREAD

**Discussion by voska7** [staxrip/staxrip#867] [Q&A] [answered]
i want to use 512kbps ac3 bitrate. but sometimes original bitrate is 640 or may be 384 bitrate. is there anyway staxrip will use 512kbps bitrate only if original bitrate is higher than 512kbps?

**Comment by jessielw** [comment, marked as answer]
You can easily see what the bitrate of the input file is before encoding. You can use copy/mux to just copy/mux the audio track over.
